install-dev/data/db_structure.sql

```diff
@@ -688,7 +688,7 @@ CREATE TABLE `PREFIX_employee` (
   `lastname` varchar(32) NOT NULL,
   `firstname` varchar(32) NOT NULL,
   `email` varchar(128) NOT NULL,
-  `passwd` varchar(32) NOT NULL,
+  `passwd` varchar(255) NOT NULL,
```

Comment:
Recommended by PHP.net, but that is if you do not lock it to `PASSWORD_BCRYPT`, like we are doing right now: http://php.net/manual/en/function.password-hash.php